policies/examples/attestation_checks.rego

```diff
@@ -0,0 +1,21 @@
+package examples.attestation_checks
+
+accepted_attestation_types := ["https://in-toto.io/Statement/v0.1"]
+
+attestation_type_ok(attestation_type) {
+	accepted_attestation_types[_] = attestation_type
+}
+
+deny = {"msg": msg} {
```

Comment:
I wonder if we should agree on the usage of `allow` vs `deny`. It matters when we query for the value. `allow` has the benefit of being possible to evaluate to a single boolean value, `true`. If it's undefined, or a different type, then it's not allowed. With `deny`, I guess we'd have to be a bit smarter around undefined values?

Reply:
Not sure, but I agree we should be thinking about this.

Some considerations:
* I expect you can derive allow from deny without much trouble, e.g. `allow { count(deny) == 0 }`.
* I think our focus is on providing useful reason messages, so whatever convention we come up with should provide a clear way to do that.